README.md

```diff
@@ -119,6 +119,7 @@ The following tables lists the configurable parameters of the Harbor chart and t
 | `database.external.username` | The username of external database | `user` |
 | `database.external.password` | The password of external database | `password` |
 | `database.external.coreDatabase` | The database used by core service | `registry` |
+| `database.external.sslmode` | Connection method of external database (require|prefer|disable) | `disable`|
```

Comment:
The default value doesn't match with the one in `values.yaml`.

Reply:
Thanks.  Corrected.